run.py

```diff
@@ -0,0 +1,4 @@
+from app import app
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
For a production environment, remove the `debug=True` value. It is recommended.
You could put this in the config file to avoid having to worrying about it.